app/code/PalashAs/AskQuestion/Cron/Changestatus.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace PalashAs\AskQuestion\Cron;
+
+use PalashAs\AskQuestion\Model\AskQuestion;
+
+class Changestatus
+{
+    protected $questionsFactory;
+
+    /**
+     * @var \Psr\Log\LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * Changestatus constructor.
+     * @param \Psr\Log\LoggerInterface $logger
+     * @param PalashAs\AskQuestion\Model\ResourceModel\AskQuestion\CollectionFactory $askQuestionsFactory
+     */
+    public function __construct(
+        \Psr\Log\LoggerInterface $logger,
+        PalashAs\AskQuestion\Model\ResourceModel\AskQuestion\CollectionFactory $askQuestionsFactory
+    )
+    {
+        $this->logger = $logger;
+        $this->questionsFactory = $askQuestionsFactory;
+    }
+
+    public function execute()
+    {
+        $questions = $this->questionsFactory->create();
+        $collection = $questions->getCollection();
+
+        foreach ($collection as $item) {
+            $item->setStatus(AskQuestion::STATUS_ANSWERED)->save();
```

Comment:
All submitted questions will be changed. Please add the filter for questions with only pending status and update time less than n days (n should be changable, constant as a simple solution)